app/src/main/java/com/example/drcreditdev/creditCardUI/creditPage/Credit_home_page.kt

```diff
@@ -1,20 +1,19 @@
-package com.example.drcreditdev.creditCard
+package com.example.drcreditdev.creditCardUI.creditPage
 
 import android.content.Intent
 import android.content.SharedPreferences
-import android.graphics.Color
 import android.graphics.Color.*
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.view.View
 import android.widget.Button
-import android.widget.EditText
 import android.widget.ImageView
 import android.widget.TextView
 import com.example.drcreditdev.R
+import com.example.drcreditdev.creditCardUI.refreshPageUI.RefreshPage
 import com.triggertrap.seekarc.SeekArc
 
-class credit_home_page : AppCompatActivity() {
+class Credit_home_page : AppCompatActivity() {
```

Comment:
ech activity name should have activity in suffix, and no _ in name

Reply:
Changes made in new commit